doc/man7/ct.pod

```diff
@@ -32,9 +32,8 @@ L<CTLOG_STORE_new(3)>,
 L<CTLOG_STORE_get0_log_by_id(3)>,
 L<SCT_new(3)>,
 L<SCT_print(3)>,
-L<SCT_verify(3)>,
 L<SCT_validate(3)>,
-L<CT_POLICY_EVAL_CTX(3)>,
```

Comment:
But it's not documented.  But CT_POLICY_EVAL_CTX_new(3) is, so I pointed to that now.